### PR TITLE
48 permanent grid

### PR DESCRIPTION
**Pull request by dhunninghake**

### CODE REVIEW

File: app/components/PermanentGallery.js [hunk 1]
```diff
@@ -1,42 +1,77 @@
 import React, { Component } from 'react';
+import ImageWrapper from './ImageWrapper';
+import { connect } from 'react-redux';
+import Atomic from '../lib/Atomic';
+
+let Masonry = require('react-masonry-component');
```
Comment (sepowitz) [inline app/components/PermanentGallery.js]: Probz best to use es6 module syntax herr

**Comment by sepowitz** [comment]
LGTM otherwise ^

File: app/components/App.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    return (
+      <div style={mainStyle}>
+        <Overlay isLoading={this.state.isLoading} />
+        <Loader isLoading={this.state.isLoading} /> 
+        {this.props.children}
+      </div>
+    )
+  }
+}
```
Comment (hhff) [inline app/components/App.js]: I don't think this file is getting used anywhere - pretty sure I deleted this in my recent PR

**Comment by dhunninghake** [comment]
nice, just got those fixes up!

**Pull request merged**